echidna/core/fill_spectrum.py

```diff
@@ -1,3 +1,4 @@
+import root_help
```

Comment:
Needs to be imported by any module that also imports ROOT (I think, was a while ago so I can't recall the reasoning).